currency/deployment/deployment.yaml

```
@@ -17,5 +17,33 @@ spec:
         - name: currency
           image: dev/currency:dev
           imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 9000
+        - name: nginx
+          image: nginx:latest
```

Best to lock this version so it won't break like Kafka did, I'll change in simple-demo
```suggestion
          image: nginx:1.27.5
```